SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -22,13 +24,30 @@ FString USpatialGDKSettings::ToString()
 	Args.Add(HeartbeatIntervalSeconds);
 	Args.Add(HeartbeatTimeoutSeconds);
 	Args.Add(ActorReplicationRateLimit);
+	Args.Add(bUsingQBI);
 
 	return FString::Format(TEXT(
 		"EntityPoolInitialReservationCount={0}, "
 		"EntityPoolRefreshThreshold={1}, "
 		"EntityPoolRefreshCount={2}, "
 		"HeartbeatIntervalSeconds={3}, "
 		"HeartbeatTimeoutSeconds={4}, "
-		"ActorReplicationRateLimit={5}")
+		"ActorReplicationRateLimit={5}, "
+		"bUsingQBI={6}")
 		, Args);
 }
+
+#if WITH_EDITOR
+// Add a pop-up to warn users to update their config upon changing the using QBI property.
+void USpatialGDKSettings::PostEditChangeProperty(struct FPropertyChangedEvent& e)
```

Comment:
Forward declaration not needed again. The argument should have the same name as in the header.